File: C++/consumo.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {

    double distancia, combustivel, consumo;

    cout << "Distancia percorrida: ";
    cin >> distancia;
    cout << "Combustivel gasto: ";
    cin >> combustivel;

    consumo = distancia / combustivel;

    cout << fixed << setprecision(3);
    cout << "Consusmo medio = " << consumo << endl;

    return 0;
}

File: Java/consumo.java
import java.util.Locale;
import java.util.Scanner;

public class Consumo {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int distancia;
        double combustivel, media;

        System.out.print("Distancia percorrida: ");
        distancia = sc.nextInt();
        System.out.print("Combustivel gasto: ");
        combustivel = sc.nextDouble();

        media = distancia / combustivel;

        System.out.println("Consumo medio = " + String.format("%.3f", media));

        sc.close();
    }
}